pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computefirewallpolicyrulefull/_http.log

```diff
@@ -644,13 +638,6 @@ X-Xss-Protection: 0
 {
   "error": {
     "code": 400,
-    "errors": [
-      {
-        "domain": "global",
-        "message": "Invalid value for field 'priority': '9000'. The firewall policy does not contain a rule at priority 9000.",
-        "reason": "invalid"
-      }
-    ],
```

Comment:
This one I think we need to fix, this error comes from getRule method, but I'm unsure why the "errors" field is missing...